app/helpers/navigation_helper.rb

```diff
@@ -7,8 +7,13 @@ def talks_link(title: I18n.t('talks.plural'))
     link_to title, talks_path
   end
 
-  def goals_link
-    link_to t('goals.plural'), goals_path
+  # Skip until 1.1 version
+  # def goals_link
+  #   link_to t('goals.plural'), goals_path
+  # end
+
+  def donate_link
+    link_to t('donations.donate'), donate_path
```

Comment:
and here I see `donate_path`. It would more consistent if you used `donations_path`